sources/src/resources/init-scripts/gradle-actions.inject-develocity.init.gradle

```diff
@@ -107,21 +108,35 @@ if (GradleVersion.current() < GradleVersion.version('6.0')) {
                     logger.quiet("Applying $BUILD_SCAN_PLUGIN_CLASS via init script")
                     applyPluginExternally(pluginManager, BUILD_SCAN_PLUGIN_CLASS)
                     if (geUrl) {
-                        logger.quiet("Connection to Develocity: $geUrl, allowUntrustedServer: $geAllowUntrustedServer")
+                        logger.quiet("Connection to Develocity: $geUrl, allowUntrustedServer: $geAllowUntrustedServer, captureTaskInputFiles: $geCaptureTaskInputFiles")
                         buildScan.server = geUrl
                         buildScan.allowUntrustedServer = geAllowUntrustedServer
                     }
                     buildScan.publishAlways()
+                    if (isAtLeast(gePluginVersion, '2.1') && GradleVersion.current() > GradleVersion.version('5.0')) {
```

Comment:
Should it really be `>`? It should skip this logic if using Gradle 5.0, but Gradle 5.1 is okay?

Reply:
You're right, 2.1 is also compatible with 5.0, not only 5.1; will change